Showcase-iOS/About/SocialMediaEndPoint.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  SocialMediaEndPoint.swift
+//  Showcase-iOS
+//
+//  Created by Lehlohonolo Mbele on 2018/05/21.
+//  Copyright © 2018 DVT. All rights reserved.
+//
+
+import Foundation
+
+enum SocialMediaEndpoint: String {
+    case website = "https://www.dvt.co.za"
```

Comment:
Can we refactor this to come from the backend in the next round, ill make a task for it. So that if any endpoints change, we will be able to change it from the backend and not have these on hardcoded on the frontend.

Reply:
@VaalSquared Please refactor this into a repo in the next commit.